package.json

```diff
@@ -119,7 +121,8 @@
     "ship": "yarn surge",
     "theme": "npx gulp less",
     "stats": "yarn build --stats && npx webpack-bundle-analyzer ./build/bundle-stats.json",
-    "watch": "node ./scripts/watch.js"
+    "watch": "node ./scripts/watch.js",
+    "postinstall": "yarn run typechain --target ethers-v5 --out-dir src/typechain src/abi/*.json src/abi/**/*.json"
```

Comment:
can't this be?
`"yarn run typechain --target ethers-v5 --out-dir src/typechain src/abi/**/*.json"`?
I am thinking that `src/abi/**/*.json` also picks up`src/abi/*.json`?

Reply:
I thought so too but that didn't work